index.bs

```diff
@@ -7,17 +7,78 @@ ED: https://webscreens.github.io/openscreenprotocol/
 Canonical URL: ED
 Editor: Mark Foltz, Google, https://github.com/mfoltzgoogle, w3cid 68454
 Repository: webscreens/openscreenprotocol
-Abstract: The Open Screen Protocol is a layered series of network protocols that allow user agents to implement the Presentation API and the Remote Playback API in an interoperable fashion.
+Abstract: The Open Screen Protocol is a suite of network protocols that allow user agents to implement the Presentation API and the Remote Playback API in an interoperable fashion.
 Group: Second Screen Community Group
 Mailing List: public-webscreens@w3c.org
 Mailing List Archives: https://lists.w3.org/Archives/Public/public-webscreens/
-Markup Shorthands: markdown yes
+Markup Shorthands: markdown yes, dfn yes, idl yes
 </pre>
 
 <p boilerplate="copyright">
 <a href="http://www.w3.org/Consortium/Legal/ipr-notice#Copyright">Copyright</a> © [YEAR] <a href="https://www.w3.org/"><abbr title="World Wide Web Consortium">W3C</abbr></a><sup>®</sup> (<a href="http://www.csail.mit.edu/"><abbr title="Massachusetts Institute of Technology">MIT</abbr></a>, <a href="http://www.ercim.eu/"><abbr title="European Research Consortium for Informatics and Mathematics">ERCIM</abbr></a>, <a href="http://www.keio.ac.jp/">Keio</a>, <a href="http://ev.buaa.edu.cn/">Beihang</a>). W3C <a href="https://www.w3.org/Consortium/Legal/ipr-notice#Legal_Disclaimer">liability</a>, <a href="https://www.w3.org/Consortium/Legal/ipr-notice#W3C_Trademarks">trademark</a> and <a href="https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document">permissive document license</a> rules apply.
 </p>
 
+<!-- TODO: Add short names to Presentation API spec, so that BS autolinking works as designed. -->
+<!-- TODO: Can autolinks to HTML51 be automatically generated? -->
+<pre class="anchors">
+urlPrefix: https://w3c.github.io/presentation-api/#dfn-; type: dfn; spec: PRESENTATION-API
+    text: controller
+    text: controlling user agent
+    text: controlling browsing context
+    text: presentation
+    text: presentation display
+    text: presentation display availability
+    text: presentation id
+    text: presentation request url
+    text: receiver
+    text: receiving browsing context
+    text: receiving user agent
+urlPrefix: https://w3c.github.io/presentation-api/; type: interface; spec: PRESENTATION-API
+    text: PresentationConnection
+urlPrefix: https://w3c.github.io/remote-playback/#dfn-; type: dfn; spec: REMOTE-PLAYBACK
+    text: remote playback device
+urlPrefix: https://www.w3.org/TR/html51/single-page.html; type: dfn; spec: HTML51
+    text: media element
+</pre>
+
+<pre class="biblio">
```

Comment:
This section should not be needed. Bikeshed fetches data from Specref, which already has entries for "PRESENTATION-API" and "REMOTE-PLAYBACK". In other words, `[[PRESENTATION-API]]` and `[[REMOTE-PLAYBACK]]` should work out of the box.

Reply:
Thanks, removed the `biblio` section and fixed up spec autolinks throughout.